### PR TITLE
Add image resize processing

### PR DESCRIPTION
**Pull request by solomkinmv**
PR for the following issue: #15

### CODE REVIEW

File: service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageProcessorImpl.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import java.io.*;
+
+@Service
+@Slf4j
+class ImageProcessorImpl implements ImageProcessor {
+
+    @Override
+    public InputStream fitByWidth(InputStream imageInputStream, int width) {
+        log.info("Trying to fit image input stream by height {}", width);
```
Comment (bcherniakh) [inline service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageProcessorImpl.java:17]: Trying to fit image input stream by height {}", sounds confusing

File: service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageServiceImpl.java [hunk 2]
```diff
@@ -20,15 +20,18 @@
 
 @Service
 @Slf4j
-public class ImageServiceImpl implements ImageService {
+class ImageServiceImpl implements ImageService {
 
+    private static final int IMAGE_WIDTH = 240;
```
Comment (bcherniakh) [inline service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageServiceImpl.java:25]: Shouldn't this be a setting somewhere in the database?

File: service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageProcessorImpl.java (hunk 1 above)
Comment (solomkinmv) [reply service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageProcessorImpl.java:17]: Suggestions?

_Sent from my Google Nexus 5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageServiceImpl.java (hunk 2 above)
Comment (solomkinmv) [reply service/src/main/java/io/github/solomkinmv/glossary/service/images/ImageServiceImpl.java:25]: If there would be any necessity I will extract this to properties. Currently I'm not sure about this

_Sent from my Google Nexus 5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Review by bcherniakh — APPROVED** [review]

**Pull request merged**